app/views/wiki/show.html.erb

```diff
@@ -25,19 +25,24 @@
   <div class="alert alert-info" role="alert">This wiki page hasn't been edited for a while. If you see anything that needs updating, please help out!</div>
 <% end %>
 
-<% unless @node.has_tag('sidebar:none') %>
+<% if @node.has_tag('sidebar:none') %>
+  <div class="col-md-1"></div>
+
+  <div class="col-md-8">
+<% else %>  
   <% if (@wikis.nil? && @notes.nil?) || @node.has_tag('sidebar:featured') %>
     <%= render :partial => "sidebar/featured" %>
   <% else %>
     <%= render :partial => "sidebar/related" %>
   <% end %>
-<% end %>
+
 
 <div class="col-md-9">
 
```

Comment:
Hi, if you look closely at the original issue, the `<% end %>` should go on this line, not below the `partial wiki/header` line. But you're almost there -- great work so far! 

If you can make these adjustments, I look forward to merging this in!

Reply:
Oops that's embarrassing, I added in the end tag and it should be good now. Sorry for that and thanks again.